CHANGELOG-1.2.md

```diff
@@ -0,0 +1,92 @@
+# Release notes for 1.2.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since 1.1.0
+
+## Changes by Kind
+
+### Other (Cleanup or Flake)
+
+- Fix a bug that when CSI migration is enabled and PV is using GA topology label, resizing is not work. ([#139](https://github.com/kubernetes-csi/external-resizer/pull/139), [@Jiawei0227](https://github.com/Jiawei0227))
+
+### Feature
+
+- Add a feature-gate to allow users to restore size of deleted PVCs ([#140](https://github.com/kubernetes-csi/external-resizer/pull/140), [@sunpa93](https://github.com/sunpa93))
```

Comment:
Does this have a k8s version dependency?

Reply:
It does have a dependency on 1.21 version.

Reply:
added a note in readme about version dependency. Good catch. We could update the version table of external-resizer but this is alpha feature, so I am not entirely sure.